pkg/util/util.go

```diff
@@ -8,12 +8,16 @@ const ExtensionAPIServerAuthenticationConfigMap = "extension-apiserver-authentic
 const RequestHeaderClientCAFileKey = "requestheader-client-ca-file"
 const VirtShareDir = "/var/run/kubevirt"
 const VirtPrivateDir = "/var/run/kubevirt-private"
+const NetworkInfoDir = VirtPrivateDir + "/network-info-cache"
```

Comment:
ha, so it doesn't matter that we have cli flags apparently if we're hardcoding values here. The VirtPrivateDir is hardcoded as well. 

This isn't something you need to deal with in this PR. Let's just remove the new network related cli option and keep it hardcoded based on whatever the VirtPrivateDir is.